crates/pathfinder/src/core.rs

```diff
@@ -369,3 +369,71 @@ impl std::fmt::Display for Chain {
         }
     }
 }
+
+/// Adds a thin Debug and Disply implementations.
+///
+/// Debug impl produces `WrapperType(hash)` instead of automatic derives generated
+/// `WrapperType(StarkHash(hash))`.
+///
+/// Display impl produces just `hash`.
+macro_rules! thin_starkhash_debug_display {
```

Comment:
Since its also used for more than just `StarkHash` wrappers.
```suggestion
macro_rules! thin_newtype_debug_display {
```

Reply:
Aah nice I just noticed it was by mistake used for `StarknetProtocolNonce` :)

Reply:
Restored StarknetProtocolVersion. I'm pretty sure there aint others.